### PR TITLE
Stops Coroners from barfing while doing autopsies...

### PR DESCRIPTION
**Pull request by SamHPurp**
Just like the detective, maybe the coroner shouldn't have the stomach of a 12 year old girl.

:cl: Purpose2
Add: Coroners are more battlehardened, and thus don't throw up around dead bodies anymore.
/ :cl:

### CODE REVIEW

**Comment by tigercat2000** [comment]
Can't they use sterile masks to avoid this?

**Comment by SamHPurp** [comment]
@tigercat2000 if they can, it wasn't working in a round or two ago, where I played Coroner.

**Comment by taukausanake** [comment]
> Shouldn't they use sterile masks to avoid this?

Fixed. But yeah, this might be fine for them either way. I support the PR

**Comment by MrFroztee** [comment]
Somebody should definitely check on the medical mask check, because I think that's another solution to such a problem.

**Comment by KasparoVy** [comment]
Sterile masks and internals work for me, but this makes sense.

**Comment by Aurorablade** [comment]
Nah I would actually prefer them not being barf immune

**Comment by Fox-McCloud** [comment]
Did you have your mask pushed down? Cause that absolutely 100% matters.

**Comment by alexkar598** [comment]
Do they spawn with the mask?

**Comment by Anticept** [comment]
The detective has an iron gut gene.

Coroners being barf immune seems fair as well.

**Comment by KasparoVy** [comment]
Plasteel*

File: code/modules/mob/living/carbon/human/life.dm [hunk 1]
```diff
@@ -1100,7 +1100,7 @@
 			if(H.species && H.species.flags & NO_BREATHE)
 				return //no puking if you can't smell!
 			// Humans can lack a mind datum, y'know
-			if(H.mind && H.mind.assigned_role == "Detective")
+			if(H.mind && (H.mind.assigned_role == "Detective" || H.mind.assigned_role == "Coroner") )
```
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/life.dm]: No spaces between the ending `)` please.

**Pull request merged**